### PR TITLE
Don't blow up when there is no data passed to the DataTable

### PR DESCRIPTION
**Pull request by wdoug**

### CODE REVIEW

File: lib/components/DataTable.js [hunk 1]
```diff
@@ -2,12 +2,16 @@ import React, {PropTypes} from 'react';
...
 export default class Admin extends React.Component {
   static propTypes = {
-    data: PropTypes.arrayOf(PropTypes.object).isRequired,
+    data: PropTypes.arrayOf(PropTypes.object),
     showHeaders: PropTypes.bool
   }
 
   render() {
     const {data} = this.props;
+    if (!data || data.length < 1) {
```
Comment (dviramontes) [inline lib/components/DataTable.js:11]: doesn't this belong in a `componentWillReceiveProps(){}`?
Then again, maybe it's a thing we can do at the router later.

Comment (dviramontes) [inline lib/components/DataTable.js:11]: works for now

**Comment by dviramontes** [comment]
:+1:

File: lib/components/DataTable.js (hunk 1 above)
Comment (wdoug) [inline lib/components/DataTable.js:11]: While eventually this whole thing should probably be redesigned, for the moment `componentWillReceiveProps` [doesn't fire when the component first renders](https://facebook.github.io/react/docs/component-specs.html#updating-componentwillreceiveprops), so you'd have to also include it in `componentDidMount`. The current implementation is simpler.

**Pull request merged**